frontend/apps/marketing/src/components/common/entries/ImageAssetEntry.ts

```diff
@@ -0,0 +1,19 @@
+import {BaseEntry} from 'contentful';
+
+export type ImageAssetEntry = BaseEntry & {
```

Comment:
This should be moved to `src/components/image` as it relates to the structure of that component. Additionally, we are enhancing the type in a generic manner so we can use a wrapper type:

```
type ContentfulEntry<ContentfulComponentProps> = BaseEntry & {
  fields :  ContentfulComponentProps
}

type ImageAssetEntry = ContentfulEntry<ImageAsset>;
```

Reply:
These aren't common types though, we can move them to a `src/types/contentful` if that helps organize it better. The generic wrapper doesn't exist yet, I'm suggesting we add it in

Reply:
Updated